src/Microsoft.Health.Fhir.SqlServer/Resources.resx

```diff
@@ -158,4 +158,7 @@
   <data name="ExecutionTimeoutExpired" xml:space="preserve">
     <value>The exection timeout expired from SQL Server.</value>
   </data>
+  <data name="GetRecordsAsyncOperationIsCanceled" xml:space="preserve">
+    <value>The operation to get resource changes has been canceled</value>
```

Comment:
nit: The period (.) is missing at the end of the sentence.

Reply:
added